CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (deps) [#1477] Migrate to CometBFT.
 - (x/incentive) [#1512] Add grpc query service.
 - (deps) [#1544] Bump confio/ics23/go to v0.9.0, cosmos/keyring to v1.2.0.
+- (x/committee) [#1562] Add CommunityPoolLendWithdrawalPermission
```

Comment:
don't forget to add the link reference at the bottom of the file 😃 should be easy to copy paste

Reply:
haha totally missed that these were linked below, lol. thanks!